simplenote.el

```diff
@@ -368,6 +368,8 @@ setting."
     ;; Don't switch mode when set via file cookie
     (when (eq major-mode (default-value 'major-mode))
       (funcall simplenote-notes-mode))
+    ;; Rename the buffer something useful
+    (rename-buffer (simplenote2--note-headline (buffer-string)))
```

Comment:
I'm sorry, that's my mistake. I had both open in tabs and submitted the PR to the wrong one. Thanks!